src/Apps/Order/Utils/currencyUtils.ts

```diff
@@ -5,7 +5,8 @@ export const appendCurrencySymbol = (
   const pricePresent: boolean =
     typeof price === "string" || typeof price == "number"
   if (currency === "USD" && pricePresent) {
-    return "US" + price
+    const parts = String(price).split("Under ")
+    return parts.length > 1 ? "Under US" + parts[1] : "US" + parts[0]
```

Comment:
I kinda wish we would get rid of the "US" prefix at checkout, like we did in the order confirmation emails, since there is only one $ currency possible at this point.

Reply:
I'm cool with that but it's not what the card is asking. I'm wondering how much it's adding to anything but it is what it is for now. Happy to remove it if you think nobody will notice. Then we can just display straight `price` here....